.travis.yml

```
@@ -4,17 +4,30 @@ node_js:
   - "6.10"
   - "lts/*"
 
+env:
+  global:
+    - PERCY_ENABLE=0
+
 matrix:
   include:
     - node_js: "11"
       env: DEPLOY=run
 
+addons:
+  chrome: stable
+
 cache:
   directories:
     - "node_modules"
 
+before_script:
+  - if [ $TRAVIS_PULL_REQUEST = 'true' ] && [ $DEPLOY = 'run' ]; then echo "Percy Enabled " && export PERCY_ENABLE=1; fi
```

Try that:
```suggestion
  - if [ $TRAVIS_PULL_REQUEST != 'false' ] && [ $DEPLOY = 'run' ]; then echo "Percy Enabled" && export PERCY_ENABLE=1; else echo "Percy Disabled" && export PERCY_ENABLE=0; fi
```
Because:
> `TRAVIS_PULL_REQUEST` is set to the pull request number if the current job is a pull request build, or `false` if it’s not.
https://docs.travis-ci.com/user/environment-variables/#convenience-variables


And I think you can remove the `global` percy variable